### ISSUE TITLE
Recovery container doesnt let me update my dockerfile Permission Denied

### ISSUE THREAD

**Issue by Dzivo** [microsoft/vscode-remote-release#7509]
VS Code
Version: 1.73.1 (user setup)
Commit: 6261075646f055b99068d3688932416f2346dd3b
Date: 2022-11-09T04:27:29.066Z (2 days ago)
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: No
Remote Development Extension Version v0.23.0

Recovery container doesnt let me update my dockerfile or any files for that matter I get permission denied.

```
Failed to save 'Dockerfile': Unable to write file 'vscode-remote://dev-container+7b227265706f7369746f727950617468223a2268747470733a2f2f656e756d736f667477617265406465762e617a7572652e636f6d2f656e756d736f6674776172652f57696e64776172642f5f6769742f57696e6477617264222c22766f6c756d654e616d65223a227673632d72656d6f74652d636f6e7461696e6572732d77696e6477617264222c22666f6c646572223a2257696e6477617264222c22696e7370656374566f6c756d65223a747275657d/workspaces/Windward/Dockerfile' (NoPermissions (FileSystemError): Error: EACCES: permission denied, open '/workspaces/Windward/Dockerfile')
```

This is crutial so that I can rebuild my container after something in dockerfile fails

**Comment by Dzivo** [comment]
@chrmarti is it duplicate ? Why is it marked as complete ? Do you need any more info ?

**Comment by chrmarti** [comment]
This fix is available in Dev Containers 0.264.0-pre-release. Let me know if that fixes this issue for you. Thanks!

(I closed the issue when I pushed the fix.)
